### ISSUE TITLE
Счетчик Матрица

### ISSUE THREAD

**Issue by DaN66O** [denpamusic/homeassistant-dlms-cosem#2]
Good afternoon!
Will this component work with "Матрица" counter ?
Can I have some details on the setup ?
Thank you in advance.

**Comment by IncognitoMSK** [comment]
Try to connect his protocol one should work

**Comment by denpamusic** [comment]
Hi,

Please note that this integration is [WIP](https://guidelines.denpa.pro/stability#work-in-progress) and is not yet ready for the public use, hence no documentation available.

This integration was initially written for a friend and is currently confirmed working only with Incotex and Iskraemeco energy meters, although it should work with any other meter that follows DLMS/COSEM standard.

It is tested with **TTL to IR probe** https://nl.aliexpress.com/item/1005004918762123.html connected to ESP32 running esphome with [esphome streaming server](https://github.com/oxan/esphome-stream-server). Any other connection method is untested and probably will not work correctly.

There's also some licensing issue with underlying library being licensed under BSL.

To sum up all of the above points, please consider waiting until the project reaches at least [Experimental](https://guidelines.denpa.pro/stability#experimental) stability state once all the problems has been solved.

Thank you!

--
Best Regards,
Denis Paavilainen
